### ISSUE TITLE
Text is very blurry

### ISSUE THREAD

**Issue by TooManyDots** [OurMachinery/themachinery-public#1254]
- **Version:** Latest Binary Release (2022.3)
- **OS:** Windows 10
![image](https://user-images.githubusercontent.com/35289254/161553174-8f86aa35-1884-482d-b34f-366a122b0919.png)

Graphics Card: NVIDIA GeForce RTX 2080

Windows Resolution:
![image](https://user-images.githubusercontent.com/35289254/161553339-578b6918-2aaa-411f-b94b-4a7db05defce.png)

**Comment by TooManyDots** [comment]
Yeah i increased the scale to make it more bearable. On a resolution scale of 1 it looks even worse.
![scale 1](https://user-images.githubusercontent.com/35289254/161574312-69d12e31-a472-4f6a-a16f-9305e1206075.png)

**Comment by tobias-ourmachinery** [comment]
![image](https://user-images.githubusercontent.com/26548681/161575975-61a7015c-439f-4c91-ac40-966713c44479.png)

What you see looks pretty similar to what I have on my low-dpi display (if you look at the screenshot make sure its not rescaled by the browser). 

With that said, I 100% agree that we probably should investigate doing something to achieve higher-quality text rendering on low-dpi displays. I think it wouldn't be too hard to add support for e.g, [subpixel rendering](https://en.wikipedia.org/wiki/Subpixel_rendering) of glyphs in the cases where we know we don't need rotated glyphs. 

If you compare The Machinery to other apps that doesn't support subpixel rendering, (e.g. the Spotify client or Blender) I think the results are equally crappy. At best some of them do some kind of stem hinting.
